query-node/mappings/src/content/channel.ts

```diff
@@ -86,14 +87,17 @@ import {
   Content_ChannelRewardClaimedAndWithdrawnEvent_V1001 as ChannelRewardClaimedAndWithdrawnEvent_V1001,
   Content_ChannelRewardUpdatedEvent_V2001 as ChannelRewardUpdatedEvent_V2001,
   Content_ChannelUpdatedEvent_V1001 as ChannelUpdatedEvent_V1001,
+  Content_ChannelUpdatedEvent_V2002 as ChannelUpdatedEvent_V2002,
   Content_ChannelVisibilitySetByModeratorEvent_V1001 as ChannelVisibilitySetByModeratorEvent_V1001,
 } from '../../generated/types'
 
 export async function content_ChannelCreated(ctx: EventContext & StoreContext): Promise<void> {
-  const { store, event } = ctx
+  const { store, event, block } = ctx
+  const { specVersion } = block.runtimeVersion
+
   // read event data
   const [channelId, { owner, dataObjects, channelStateBloatBond }, channelCreationParameters, rewardAccount] =
-    new ChannelCreatedEvent_V1001(event).params
+    specVersion < 2002 ? new ChannelCreatedEvent_V1001(event).params : new ChannelCreatedEvent_V2002(event).params
```

Comment:
It doesn't look like the `ChannelCreated` event was changed in this PR.. or `nara`, So why is this needed? Is it related possible to `1001.json` being modified? Perhaps since work on crt was started such a long time ago, the spec version was 1001 at that time..?

Reply:
fixed by copying 1001.json and 2001.json from nara https://github.com/Joystream/joystream/pull/4871/commits/ead0209600d711681888c6aff1ed222e07422b4e

Reply:
Ok, but technically the code now is suggesting that there is a difference in types. Its not a big deal but it might confuse the reader.